AlphaWallet/Configuration.swift

```diff
@@ -0,0 +1,98 @@
+            }
+            static let defaultForegroundText = UIColor { trait in
+                return colorFrom(trait: trait, lightColor: R.color.black()!, darkColor: R.color.white()!)
+            }
+
+            static let primaryButtonBackgroundLight = R.color.cod()!
+            static let primaryButtonBackgroundDark = R.color.alabaster()!
+            static let primaryButtonHighlightedBackground = UIColor { trait in
+                return colorFrom(trait: trait, lightColor: R.color.black()!, darkColor: R.color.concrete()!)
+            }
+            static let primaryButtonBackgroundInactive = UIColor { trait in
+                return colorFrom(trait: trait, lightColor: R.color.mike()!, darkColor: R.color.mine()!)
+            }
+            static let primaryButtonBorderInactive = UIColor { trait in
+                return colorFrom(trait: trait, lightColor: R.color.mike()!, darkColor: R.color.mine()!)
+            }
+            static let primaryButtonTextActive = UIColor { trait in
+                return colorFrom(trait: trait, lightColor: R.color.white()!, darkColor: R.color.black()!)
+            }
+            static let primaryButtonTextInactive = UIColor { trait in
+                return colorFrom(trait: trait, lightColor: R.color.white()!, darkColor: R.color.black()!)
+            }
+            static let primaryButtonBorderActive = UIColor { trait in
+                return colorFrom(trait: trait, lightColor: R.color.cod()!, darkColor: R.color.alabaster()!)
+            }
+
+            static let secondaryButtonBackgroundLight = R.color.white()!
+            static let secondaryButtonBackgroundDark = R.color.dusty()!
+            static let secondaryButtonHighlightedBackground = UIColor { trait in
+                return colorFrom(trait: trait, lightColor: R.color.concrete()!, darkColor: R.color.dusty()!)
+            }
+            static let secondaryButtonBackgroundInactive = UIColor { trait in
+                return colorFrom(trait: trait, lightColor: R.color.mike()!, darkColor: R.color.dusty()!)
+            }
+            static let secondaryButtonBorderInactive = UIColor { trait in
+                return colorFrom(trait: trait, lightColor: R.color.mike()!, darkColor: R.color.dusty()!)
+            }
+            static let secondaryButtonTextActive = UIColor { trait in
+                return colorFrom(trait: trait, lightColor: R.color.black()!, darkColor: R.color.white()!)
+            }
+            static let secondaryButtonTextInactive = UIColor { trait in
+                return colorFrom(trait: trait, lightColor: R.color.black()!, darkColor: R.color.white()!)
+            }
+            static let secondaryButtonBorderActive = UIColor { trait in
+                return colorFrom(trait: trait, lightColor: R.color.white()!, darkColor: R.color.dusty()!)
+            }
+        }
+    }
+}
+
+class UIKitFactory {
+
+    static func defaultView(autoResizingMarkIntoConstraints: Bool = false) -> UIView {
```

Comment:
How about rename like these:

```
`defaultView(withAutoResizingMarkIntoConstraints:)` -> `createView(withAutoResizingMarkIntoConstraints:)`
`defaultLabel(text: String)` -> `createLabel()`
`decorateAsDefaultView(_:)` -> `configure(view:)`
`decorateAsDefaultLabel(_:)` -> `configure(label:)`
```

And maybe the last 2 should be private?

Reply:
The last 2 are not private because I use them to change the configuration of views constructed in other methods; I didn't want to change all the code.

In the future, when a uiview is required, I would recommend that it be vended from UIKitFactory. We can set the font, background color etc all in one spot.

Reply:
I'm thinking of adding styles to these calls. Similar to the ones Apple is using for UILabels.

Reply:
Not sure what you mean, but let's go with it and see.

Reply:
I mean in other places, people use their own code to create and set up views. This call just replaces the required properties for those views with minimal disruption to the code.